design/func_call/func_call.md

```diff
@@ -0,0 +1,65 @@
+# Function Calls
```

Comment:
Some documentation as to which portions of the verilog is associated with `hrange` and `dup_range` would be great

Reply:
Is it now more understandable?